### ISSUE TITLE
2.0.8 Wireless Carplay does not Work after Userspace Reboot

### ISSUE THREAD

**Issue by BBBKyleBBB** [opa334/Dopamine#453]
Hey Opa, before I start, thanks for all your work on this latest jailbreak.  In Dopamine 2.0.6, I'm able to connect to wireless carplay and the connection will reestablish when necessary after a Userspace Reboot in the Dopamine application.  Wireless carplay would not connect at all on Dopamine 2.0.7, and the initial connection was fixed on Dopamine 2.0.8.  However, when I trigger a Userspace Reboot on 2.0.8, I am unable to connect to wireless carplay afterwards.

**Comment by opa334** [comment]
What device? What version?...

**Comment by BBBKyleBBB** [comment]
Ah my bad, iPhone 8, ios version 16.6.1

**Comment by TheGamingGallifreyan** [comment]
I use wireless CarPlay every day and have not had any issues with it related to Dopamine.

I've even userspace rebooted while its connected and it reconnects almost instantly.

Do you have the same problem with tweak injection disabled? I had a lot of issues with CarPlay with the Airaw tweak installed.

iPhone 14 Pro Max 16.4.1

**Comment by opa334** [comment]
Okay so if there is a problem, it most likely only affects arm64. So it works after the phone is jailbroken (e.g. after Dopamine has rebooted the userspace) but if you trigger another userspace reboot yourself later, it breaks it?

**Comment by BBBKyleBBB** [comment]
Ok it turns out I may just have been in a deadzone of some kind networking wise, I can confirm that in my usual areas, I'm able to use wireless carplay like expected when using Dopamine 2.0.8.  Sorry for opening this issue unnecessarily!

**Comment by BBBKyleBBB** [comment]
Ok, I tested a bit more on 2.0.8, and I closed the issue yesterday after jailbreaking on Dopamine 2.0.8 and connecting to wireless Carplay for the first time.  However after disconnecting from wireless Carplay on 2.0.8, I was not able to reconnect, even if forgetting the phone from the vehicle and the vehicle from the phone, and connecting them both again.  The issue does still occur if tweak injection is disabled in Dopamine 2.0.8, and if the jailbreak is hidden.  For some reason, 2.0.6 allows me to continually connect and reconnect to wireless Carplay, while 2.0.8 does not.
